docs/sources/reference/commandline/cli.md

```diff
@@ -17,6 +17,30 @@ or execute `docker help`:
 
       ...
 
+## Environment Variables
+
+For easy reference, the follow list of environment variables are supported by
```

Comment:
You still have a typo here! follow-->following.

 Also, "are" should be "is" (list is singular). Or you could just get rid of the "list of". Sorry if I introduced that error!